go/host/db/blocks.go

```diff
@@ -27,13 +28,39 @@ func (db *DB) AddBlockHeader(header *types.Header) error {
 		return fmt.Errorf("could not write block header. Cause: %w", err)
 	}
 
+	// Update the head if the new height is greater than the existing one.
+	tipBlockHeader, err := db.GetTipBlockHeader()
```

Comment:
Tip is also a common name, although we don't use it in our code base much.
Tip is better than Head because of Header. `GetTipBlockHeader` vs `GetHeadBlockHeader`.

I think if we use Tip instead of Head it's clearer. I'm also ok with having these mixed atm.

Reply:
Yeah fair, maybe I've been here too long, have gotten used to the concept of HeadHeaders haha.